readthedocs/config/config.py

```diff
@@ -271,10 +286,9 @@ def validate_output_base(self):
         """Validates that ``output_base`` exists and set its absolute path."""
         assert 'output_base' in self.env_config, (
                '"output_base" required in "env_config"')
-        base_path = os.path.dirname(self.source_file)
         output_base = os.path.abspath(
             os.path.join(
-                self.env_config.get('output_base', base_path),
+                self.env_config.get('output_base', self.base_path),
```

Comment:
This is from `validate_base` we should remove that and `validate_name`, we never used that, probably another PR.